x-pack/plugins/maps/public/components/layer_panel/join_editor/view.js

```diff
@@ -16,8 +16,9 @@ import {
 } from '@elastic/eui';
 
 import { Join } from './resources/join';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
 
-export function JoinEditor({ joins, layer, onChange }) {
+export const JoinEditor = injectI18n(({ joins, layer, onChange, intl }) => {
```

Comment:
@Bamieh just to confirm, but this construct is no longer required, correct?

I should just use `i18n.translate` instead?

Reply:
Yes we are pushing towards removing all `injectI18n` occurrences from the Kibana codebase and use `i18n.translate` instead.